lochness/rpms/__init__.py

```diff
@@ -329,6 +362,7 @@ def sync(Lochness, subject, dry=False):
                                BIDS=Lochness['BIDS'])
         proc_dst = Path(proc_folder) / f"{subject_id}_{measure}.csv"
 
+        get_run_sheets_for_datatypes(target_df_loc)
```

Comment:
Were we not extracting RPMS run sheets so far?

Reply:
We have been extract the run sheets from RPMS for a long time, but the commits and push were delayed on my end. Thanks.